content/news/2020-02-13_google_code-in_2019_finished.md

```diff
@@ -0,0 +1,30 @@
++++
+type = "blog"
+title = "Google Code-in 2019 finished"
+author = "humdinger"
+date = "2020-02-13 08:14:36+01:00"
+tags = ["gci", "code-in"]
++++
+
+This 2019/2020 Google Code-in (GCI) was the 10th iteration in as may years and according to [Google's stats](https://opensource.googleblog.com/2020/02/announcing-our-google-code-in-2019.html) it was the most successful yet: In 7 weeks 3,566 students from 76 countries finished 20,840 tasks for 29 open source organisations!
+
+Haiku was one of those organisations and we had our share of dedicated students that completed numerous tasks, big and small. You may have noticed the increased updates on the [HaikuPorts commits](https://github.com/haikuports/haikuports/commits/master)  page, that got even more busy than usual. Other tasks resulted in new and updated documentation, guides and videos. Projects at [HaikuArchives](https://github.com/HaikuArchives) saw bugfixes and new features. Apps were tested and new issues were discovered and filed at their bugtracker.
+
+Of all our students our mentors pick two Grand Prize Winners who - along with their parents/guardians - are invited to the Google HQ in California. Picking the winners isn't easy, as you can imagine. It's not so much the number of tasks (though both of them have about 30, usually 10+ tasks get you into that judging round), but more the difficulty of the tasks, the quality of the solutions and how self-sufficient the students work.
+
+Haiku's Grand Prize Winners this year are:
+
+* Abdur-Raheem Idowu (R4H33M) from Norway
+* Rafał Bernacki (RaV) from Poland
+
+The Runners Up, who are the understudies should anything prevent a Grand Prize Winner to follow Google's invitation:
```

Comment:
I think that information is not published on the official GCI results, so maybe we should put all 4 as finalists?

Reply:
The Goolge blog linked at the start has that distinction, Winners/Runners up/Finalists:
https://opensource.googleblog.com/2020/02/announcing-our-google-code-in-2019.html